app/ts/components/pages/PersonalSign.tsx

```diff
@@ -309,48 +317,71 @@ function GnosisSafeExtraDetails({ visualizedPersonalSignRequestSafeTx, renameAdd
 				</>
 				: <></>
 			}
-			<CellElement text = 'baseGas: '/>
+			<CellElement text = 'Base Gas: '/>
 			<CellElement text = { visualizedPersonalSignRequestSafeTx.message.message.baseGas }/>
-			<CellElement text = 'gasPrice: '/>
+			<CellElement text = 'Gas Price: '/>
 			<CellElement text = { visualizedPersonalSignRequestSafeTx.message.message.gasPrice }/>
 			{ visualizedPersonalSignRequestSafeTx.message.message.gasToken !== 0n
 				? <>
-					<CellElement text = 'gasToken: '/>
+					<CellElement text = 'Gas Token: '/>
 					<CellElement text = { <SmallAddress addressBookEntry = { visualizedPersonalSignRequestSafeTx.gasToken } renameAddressCallBack = { renameAddressCallBack } /> }/>
 				</>
 				: <></>
 			}
-			<CellElement text = 'nonce: '/>
+			<CellElement text = 'Nonce: '/>
 			<CellElement text = { visualizedPersonalSignRequestSafeTx.message.message.nonce }/>
-			<CellElement text = 'operation: '/>
+			<CellElement text = 'Operation: '/>
 			<CellElement text = { visualizedPersonalSignRequestSafeTx.message.message.operation }/>
 			{ visualizedPersonalSignRequestSafeTx.message.message.refundReceiver !== 0n ?
 				<>
-					<CellElement text = 'refundReceiver: '/>
+					<CellElement text = 'Refund Receiver: '/>
 					<CellElement text = { <SmallAddress addressBookEntry = { visualizedPersonalSignRequestSafeTx.refundReceiver } renameAddressCallBack = { renameAddressCallBack } /> }/>
 				</>
 				: <></>
 			}
-			<CellElement text = 'safeTxGas: '/>
+			<CellElement text = 'Safe Transaction Gas: '/>
 			<CellElement text = { visualizedPersonalSignRequestSafeTx.message.message.safeTxGas }/>
-			<CellElement text = 'to: '/>
+			<CellElement text = 'To: '/>
 			<CellElement text = { <SmallAddress addressBookEntry = { visualizedPersonalSignRequestSafeTx.to } renameAddressCallBack = { renameAddressCallBack } /> }/>
-			<CellElement text = 'value: '/>
-			<CellElement text = { <Ether amount = { visualizedPersonalSignRequestSafeTx.message.message.value } rpcNetwork = { visualizedPersonalSignRequestSafeTx.rpcNetwork } fontSize = 'normal'/>  }/>
+			<CellElement text = 'Value: '/>
+			<CellElement text = { <Ether amount = { visualizedPersonalSignRequestSafeTx.message.message.value } rpcNetwork = { visualizedPersonalSignRequestSafeTx.rpcNetwork } fontSize = 'normal'/> }/>
+			<CellElement text = 'Domain Hash: '/>
+			<CellElement text = { visualizedPersonalSignRequestSafeTx.domainHash }/>
+			<CellElement text = 'Message Hash: '/>
+			<CellElement text = { visualizedPersonalSignRequestSafeTx.messageHash }/>
+			<CellElement text = 'Safe Transaction Hash: '/>
+			<CellElement text = { visualizedPersonalSignRequestSafeTx.safeTxHash }/>
```

Comment:
This should make them align (fixed width font) since they are all the same length, which IMO makes it look a bit cleaner/nicer.
```suggestion
			<CellElement text = 'Domain Hash: '/>
			<code><CellElement text = { visualizedPersonalSignRequestSafeTx.domainHash }/></code>
			<CellElement text = 'Message Hash: '/>
			<code><CellElement text = { visualizedPersonalSignRequestSafeTx.messageHash }/></code>
			<CellElement text = 'Safe Transaction Hash: '/>
			<code><CellElement text = { visualizedPersonalSignRequestSafeTx.safeTxHash }/></code>
```